recipe/meta.yaml

```diff
@@ -1,18 +1,18 @@
 {% set name = "mpltoolbox" %}
-{% set version = "24.5.1" %}
+{% set version = "25.4.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
```

Comment:
python_min needs to be set to match mpltoolbox to 3.10, the default conda-forge one is 3.9 right now. https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/a48c48d69d68fab3e041c52b9fd51a9ba0d38482/recipe/conda_build_config.yaml#L870